systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaBridgeResource.java

```diff
@@ -35,6 +35,31 @@ public static DoneableKafkaBridge kafkaBridge(String name, String clusterName, S
         return deployKafkaBridge(defaultKafkaBridge(kafkaBridge, name, clusterName, bootstrap, kafkaBridgeReplicas).build());
     }
 
+    public static DoneableKafkaBridge kafkaBridgeWithCoors(String name, String bootstrap, int kafkaBridgeReplicas,
+                                                           String allowCoorsOrigin, String allowCoorsMethods) {
```

Comment:
"Coors" as already raised by @Frawless and I would use "allowed" instead of "allow". This is valid in other parts of the code.